content/posts/fleek-network-running-a-node-in-a-docker-container.md

```diff
@@ -23,37 +23,61 @@ tags:
 
 Alternatively, if you need a deep dive into Docker, check the official getting started [here](https://docs.docker.com/get-started/).
 
 ## Topics
+  - [For the impatient](#for-the-impatient)
   - [Setup](#docker-setup)
     - [Install Docker desktop](#install-docker-desktop)
+    - [Recommended settings](#recommended-settings)
   - [Image](#image)
     - [Build the Docker image](#build-the-docker-image)
   - [Container](#container)
-    - [Run the container from official nightly image](#run-the-container-from-official-nightly-image)
+    - [Run the container from the official nightly image](#run-the-container-from-the-official-nightly-image)
     - [Run the Docker container](#run-the-docker-container)
     - [Ursa healthcheck](#ursa-healthcheck)
   - [Utility commands](#utility-commands)
     - [Restart the Docker container](#restart-the-docker-container)
     - [Delete the Docker container](#delete-the-docker-container)
     - [Execute Ursa CLI commands in the container](#execute-ursa-cli-commands-in-the-container)
     - [Execute Bash Shell in the container](#execute-bash-shell-in-the-container)
-  - [Stack](#stack)
+  - [Run the container from the recommended stack](#run-the-container-from-the-recommended-stack)
     - [Running a stack with Docker compose](#running-a-stack-with-docker-compose)
   - [Conclusion](#conclusion)
 
-As Fleek Network's repositories are in constant development and change, you should consider that the following guide was [checked in](https://git-scm.com/docs/git-checkout) to commit `60a37c62e`. While we try our best to update documentation and guides during development, there might be breaking changes which might take some time to reflect in our docs. To avoid disappointment, feel free to check into commit `60a37c62e` or contribute by getting in touch with us, or sending a PR in the relevant context 🙏.
+As Fleek Network's repositories are in constant development and change, you should consider that the following guide was [checked in](https://git-scm.com/docs/git-checkout) to commit `180585c`. While we try our best to update documentation and guides during development, there might be breaking changes that might take some time to reflect in our docs. To avoid disappointment, feel free to check into commit `180585c` or contribute by getting in touch with us, or sending a PR in the relevant context 🙏.
+
+### For the impatient 😤
+
+For the one's interested in running the node as quickly as possible, there are a few options, among those we pick two you might find handy:
+
+- A Docker compose Stack, which is an opinionated stack that provides a proxy service for IPFS, a service to enable HTTPS on your server, services for monitoring and analytics, etc. You can run a stack quickly by following the instructions in [Run the container from the recommended stack](#run-the-container-from-the-recommended-stack).
```

Comment:
`a proxy service for ursa` its not a proxy for IPFS

Reply:
sure